lms/djangoapps/mobile_api/users/views.py

```diff
@@ -258,35 +259,43 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           certified).
         * url: URL to the downloadable version of the certificate, if exists.
     """
-    queryset = CourseEnrollment.objects.all()
-    serializer_class = CourseEnrollmentSerializer
-    lookup_field = 'username'
-
-    # In Django Rest Framework v3, there is a default pagination
-    # class that transmutes the response data into a dictionary
-    # with pagination information.  The original response data (a list)
-    # is stored in a "results" value of the dictionary.
-    # For backwards compatibility with the existing API, we disable
-    # the default behavior by setting the pagination_class to None.
-    pagination_class = None
```

Comment:
Are you confident this won't break backward compatibility, as the code says it's required?  Have you tested this change against the mobile app?